Formula/ski.rb

```diff
@@ -14,15 +23,18 @@ class Ski < Formula
   end
 
   head do
-    url "https://gitlab.com/esr/ski.git"
+    url "https://gitlab.com/esr/ski.git", branch: "master"
     depends_on "xmlto" => :build
   end
 
+  uses_from_macos "python"
```

Comment:
Existing bottles should be okay with this. In next update, it may need one of following to support old (not-bottled) macOS:
- Homebrew Python3 pre-Catalina
   ```rb
   uses_from_macos "python", since: :catalina
   ```
- Homebrew Python3 everywhere
   ```rb
   depends_on "python@3.10"
   ```
- Inreplace `#!/usr/bin/env python3` to `#!/usr/bin/env python` for pre-Catalina as upstream supports both Python2 and Python3.